vendor/knative.dev/pkg/version/version.go

```diff
@@ -33,7 +33,7 @@ const (
 	// NOTE: If you are changing this line, please also update the minimum kubernetes
 	// version listed here:
 	// https://github.com/knative/docs/blob/mkdocs/docs/snippets/prerequisites.md
-	defaultMinimumVersion = "v1.19.0"
+	defaultMinimumVersion = "v1.22.0"
```

Comment:
Eventing is setting this to 1.21.0?

Reply:
4.8 is 1.21, that's why

Reply:
Ok then counted it wrongly so should this be 1.21.0 as well?

Reply:
No, this is now whatever is set in code at the current version, no more patching since `getMinimumVersion` will use the injected env variable

Reply:
Yes but for consistency reasons it would be nice to be the aligned. Even if it is not used it is an indeication of the version supported.

Reply:
Yeah maybe a note in project.yaml that this file is ginored.

Reply:
This will align itself once we remove patching from other midstream repositories, I don't think we should keep having a patch for alignment in this repo because that's defeat the purpose of the PR itself

Reply:
+1 on adding a comment if we find that useful

Reply:
When someone finds the `version.go`, I'm sure I will check if what `KUBERNETES_MIN_VERSION` is set. 

https://github.com/openshift-knative/serverless-operator/blob/5cb524816e9084acc36457df5e09c7f3c84bb5a2/vendor/knative.dev/pkg/version/version.go#L29-L36

Reply:
Anyway, I added the comment as I don't want to keep this PR open...